modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js

```diff
@@ -334,6 +335,8 @@ Liferay.Util.unescapeHTML = unescapeHTML;
 
 Liferay.Util.Cookie = Cookie;
 
+Liferay.Util.COOKIE_TYPES = COOKIE_TYPES;
```

Comment:
We are already exposing consent types, as `Liferay.Util.SessionStorage.TYPES`. See example usage [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/js/OAuth2Client.ts#L104-L107).

Reply:
Upps didn't notice that 😱